datadog_checks_dev/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # CHANGELOG - Datadog Checks Dev
 
+## 16.0.0 / 2022-02-07
+
+* [Added] Support Python checks defined by a `pyproject.toml` file. See [#11233](https://github.com/DataDog/integrations-core/pull/11233).
+* [Added] Add snmp build-traps-db command. See [#11235](https://github.com/DataDog/integrations-core/pull/11235).
+* [Added] Add curated_metric column to check validation. See [#11168](https://github.com/DataDog/integrations-core/pull/11168).
+* [Fixed] Safely check the dashboards key exists before trying to write to it. See [#11285](https://github.com/DataDog/integrations-core/pull/11285).
+* [Fixed] Validate all `curated_metric` rows and properly validate empty `metadata.csv` files. See [#11273](https://github.com/DataDog/integrations-core/pull/11273).
+* [Fixed] More specific config validation error message. See [#11272](https://github.com/DataDog/integrations-core/pull/11272).
+* [Fixed] Unpin black. See [#11270](https://github.com/DataDog/integrations-core/pull/11270).
+* [Changed] Add tls_protocols_allowed option documentation. See [#11251](https://github.com/DataDog/integrations-core/pull/11251).
```

Comment:
This should be no-changelog for datadog-checks-dev

Reply:
Fixed